### PR TITLE
SAK-43934 Add German Translation

### PR DESCRIPTION
**Pull request by birkastoana**
Translated two tools in de_DE

### CODE REVIEW

**Comment by birkastoana** [comment]
finished translation, locale de_DE can be added as a standard to current versions

File: citations/citations-util/util/src/bundle/citations.properties [hunk 1]
```diff
@@ -227,8 +227,8 @@ no.results			= No results found to match your query
 title.results		= Search Results
 add.results			= Add
 add_alt.results		= Add to citation list
-remove.results		= Remove
```
Comment (mpellicer) [inline citations/citations-util/util/src/bundle/citations.properties:230]: Can you revert these changes? You're updating the default locale.

File: config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties [hunk 2]
```diff
@@ -11,7 +11,7 @@ application/macwriteii=Mac Write
 application/mathematica=Mathematica
 application/ms-excel=Excel
 application/excel=Excel
-application/vnd.ms-excel.sheet.macroEnabled.12=Excel
+
```
Comment (mpellicer) [inline config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties:14]: Can you revert these changes? You're changing the default locale.

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties [hunk 3]
```diff
@@ -30,9 +30,9 @@ noDelegatedAccessUser=This user does not have any delegated access
 noHierarchy=The hierarchy hasn't been setup for this tool.  In order to use Delegated Access, you must create a hierarchy.  Please consult the following confluence page: https://confluence.sakaiproject.org/display/DAC/Delegated+Access+Tool
 addAccess=Add Access
 add=Add
-remove=Remove
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties:33]: Can you revert these changes? You're changing the default locale.

File: lessonbuilder/tool/src/resources/messages.properties [hunk 4]
```diff
@@ -501,7 +501,7 @@ simplepage.permissions-general=You do not have the required permissions to do th
 simplepage.comment-unknown-user=Unknown User
 simplepage.comment-you=You
 simplepage.comment-anonymous-message=Normal users can only see the Anonymous names. The commenters' actual names are shown only to you and other site administrators.
-simplepage.edit-comment=Edit
```
Comment (mpellicer) [inline lessonbuilder/tool/src/resources/messages.properties:504]: Can you revert these changes? You're changing the default locale.

File: msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties [hunk 5]
```diff
@@ -816,8 +816,8 @@ move_topics=topics
 move_cancel=Cancel
 move_move=Move Thread(s) to Selected Topic
 move_thread=Move Thread(s)
-move_thread_info1= The following conversation(s) will be moved from
-move_thread_info2= to the topic selected below.
+move_thread_info1= The following conversation(s) will be moved from\u0020
```
Comment (mpellicer) [inline msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties:819]: Can you revert these changes? You're changing the default locale.

File: samigo/samigo-pack/src/bundle/samigo-noti-prefs.properties [hunk 6]
```diff
@@ -2,6 +2,6 @@ prefs_title = Tests & Quizzes
 # Do not remove prefs_description or prefs_title_override, check SAK-21078 for risk and code for reason
 prefs_description =
 prefs_title_override =
-prefs_opt3 = Send me an email confirmation each time I submit an assessment
-prefs_opt2 = Send me one email per day confirming all of my submissions
-prefs_opt1 = Do not send me any email confirmations
+prefs_opt3 = Seperate Best\u00e4tigung f\u00fcr jede Einreichung per Email empfangen
```
Comment (mpellicer) [inline samigo/samigo-pack/src/bundle/samigo-noti-prefs.properties:5]: Can you revert these changes? You're changing the default locale.

File: site-manage/site-manage-impl/impl/src/bundle/UserAuditService.properties [hunk 7]
```diff
@@ -1,7 +1,5 @@
 #M will display as Last Name, First Name (user's eid)
 M = {0} ({1})
 S = Self
-#M will display as Last Name, First Name (user's eid)
```
Comment (mpellicer) [inline site-manage/site-manage-impl/impl/src/bundle/UserAuditService.properties:4]: Can you revert these changes? You're changing the default locale.

File: citations/citations-util/util/src/bundle/citations.properties (hunk 1 above)
Comment (birkastoana) [reply citations/citations-util/util/src/bundle/citations.properties:230]: Indeed, this is a double key value.

File: config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties (hunk 2 above)
Comment (birkastoana) [reply config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties:14]: deleted double key

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties (hunk 3 above)
Comment (birkastoana) [reply delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties:33]: double key value

File: lessonbuilder/tool/src/resources/messages.properties (hunk 4 above)
Comment (birkastoana) [reply lessonbuilder/tool/src/resources/messages.properties:504]: this was a double key

File: msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties (hunk 5 above)
Comment (birkastoana) [reply msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties:819]: done

File: samigo/samigo-pack/src/bundle/samigo-noti-prefs.properties (hunk 6 above)
Comment (birkastoana) [reply samigo/samigo-pack/src/bundle/samigo-noti-prefs.properties:5]: done

File: site-manage/site-manage-impl/impl/src/bundle/UserAuditService.properties (hunk 7 above)
Comment (birkastoana) [reply site-manage/site-manage-impl/impl/src/bundle/UserAuditService.properties:4]: these were double keys

File: config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties (hunk 2 above)
Comment (mpellicer) [reply config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties:14]: good

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties (hunk 3 above)
Comment (mpellicer) [reply delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties:33]: Good

File: lessonbuilder/tool/src/resources/messages.properties (hunk 4 above)
Comment (mpellicer) [reply lessonbuilder/tool/src/resources/messages.properties:504]: good thanks

**Review by SedueRey — COMMENTED** [review]
File GradebookNgApplication_de_DE.properties isn't where it belongs, it is on 
sakai/gradebookng/tool/src/java/org/sakaiproject/gradebookng/, wrong place. @mpellicer what do you think?
